docs/get-started/connect/mainnet.md

```diff
@@ -82,10 +82,16 @@ teku \
     --ee-endpoint=http://localhost:8551          \
     --ee-jwt-secret-file=<path to jwtsecret.hex> \
     --metrics-enabled=true                       \
-    --rest-api-enabled=true
+    --rest-api-enabled=true                      \
+    --checkpoint-sync-url=<checkpoint state URL>
```

Comment:
Let's use `<checkpoint sync URL>` for the placeholder to avoid any confusion with users. That applies to the other examples as well.